src/main/java/dinkplugin/notifiers/QuestNotifier.java

```diff
@@ -5,15 +5,32 @@
 import dinkplugin.util.QuestUtils;
 import dinkplugin.util.Utils;
 import dinkplugin.notifiers.data.QuestNotificationData;
+import net.runelite.api.VarPlayer;
+import net.runelite.api.annotations.Varbit;
 import net.runelite.api.events.WidgetLoaded;
 import net.runelite.api.widgets.Widget;
 import net.runelite.api.widgets.WidgetInfo;
+import net.runelite.client.callback.ClientThread;
 import org.apache.commons.lang3.StringUtils;
+import org.jetbrains.annotations.VisibleForTesting;
+
+import javax.inject.Inject;
 
 import static net.runelite.api.widgets.WidgetID.QUEST_COMPLETED_GROUP_ID;
 
 public class QuestNotifier extends BaseNotifier {
 
+    @Varbit
+    @VisibleForTesting
+    static final int COMPLETED_ID = 6347, TOTAL_ID = 11877; // https://github.com/Joshua-F/cs2-scripts/blob/master/scripts/%5Bproc,questlist_completed%5D.cs2#L5
+
+    @Varbit
+    @VisibleForTesting
+    static final int QP_TOTAL_ID = 1782; // https://github.com/Joshua-F/cs2-scripts/blob/master/scripts/%5Bproc,questlist_qp%5D.cs2#L5
```

Comment:
nitpick: Could the comment be on as separate line? would make commit history slightly cleaner if we have to update one or the other